### ISSUE TITLE
Feature request : Add templates for specific logins, credit cards 

### ISSUE THREAD

**Issue by F3000** [strongbox-password-safe/Strongbox#281]
E.g. a user can click on template “MasterCard” and see a form with pre added fields for all information on a real card . 

The user only have to fill out the personal data from his card and save the entry . 

The same for Amex/Driving license etc...

**Comment by Slummi** [comment]
This could be implemented in the same way as the KeePass plugin [KPEntryTemplates](https://github.com/mitchcapper/KPEntryTemplates) does.

**Comment by F3000** [comment]
> This could be implemented in the same way as the KeePass plugin [KPEntryTemplates](https://github.com/mitchcapper/KPEntryTemplates) does.

Good Idea . I. Think the editor in iOS (for all users) would be to heavy but a pre defined grid from Marc would be cool .

**Comment by strongbox-mark** [comment]
Will wait on KeePassXC implementation is done:

https://github.com/keepassxreboot/keepassxc/issues/863

**Comment by F3000** [comment]
Ok . What do you think , did the other team implement in in the near future or something we have to wait some years ?

**Comment by strongbox-mark** [comment]
Not sure how long it will take but I know it's pretty high priority for them so I don't think it will be too long.

**Comment by federicocappelli** [comment]
FYI: KeePass conversation continues here: https://github.com/keepassxreboot/keepassxc/issues/8228 Scheduled for v2.8.0

**Comment by sammcj** [comment]
Would this allow Strongbox to autofill name/address/credit card etc... fields? (Currently my largest pain point), if so - some overlap with https://github.com/strongbox-password-safe/browser-autofill/issues/6

**Comment by strongbox-mark** [comment]
Hi @sammcj - Yes, that's the idea, we need the entry type/template support first, then we can offer in Browser plugin

**Comment by a-b** [comment]
Finding a UX solution to display the expiration date in MM/YY format would be helpful.

**Comment by justinhorner** [comment]
Any update on this?  

I came across this because I was going to submit a 'bug' because if one is traveling across time zones, the expiration date can be 'modified' in Strongbox resulting in incorrect expiration dates (because you have to select a time, not just a date) and in turn invalid payment information on purchases.  

As @a-b points out, it would be nice to have MM/YY format or even just the ability to have a Date field, rather than a DateTime.  I can definitely see the value in a DT in certain circumstances but in cases like a credit card, it can actually result in a negative UX experience.

Thanks for the great app!

**Comment by jonaswouters** [comment]
> Not sure how long it will take but I know it's pretty high priority for them so I don't think it will be too long.

4 years later, and still no progress 😢

**Comment by MattTheBanana** [comment]
Happy 2026, y'all! Just joining the community to come across the same pain point :P
